wpeverest/user-registration
# Feature - Ajax form submission on edit profile

### All Submissions:






### Changes proposed in this Pull Request:
Previously conditional rules were not being parsed at all in the edit profile section and were showing a conditionally hidden field to be required. This PR mitigates this issue by introducing ajax form submission on edit-profile form in order properly parse the form data.

### How to test the changes in this Pull Request:

1. Enable ajax form submission on edit profile option from User Registration Settings -> General Options -> My Account Section.
2. Navigate to edit profile tab in my account pages, edit some of the user details and do update conditional fields values also.
3. Submit the form, the form will be submitted using ajax call and verify if the conditional fields are working perfectly.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Ajax form submission on edit profile.
